api.js

```diff
@@ -13,3 +13,17 @@ export function getComments(id) {
     `https://nc-news-by-amir.onrender.com/api/articles/${id}/comments`
   );
 }
+
+export function upvoteArticle(id) {
+  return axios.patch(
+    `https://nc-news-by-amir.onrender.com/api/articles/${id}/`,
+    { inc_votes: 1 }
+  );
+}
+
+export function downoteArticle(id) {
+  return axios.patch(
+    `https://nc-news-by-amir.onrender.com/api/articles/${id}/`,
+    { inc_votes: -1 }
+  );
```

Comment:
These 2 functions could potentially be refactored to just be one. You'd then send either 1 or -1 to the function 👍